src/displayapp/screens/Dice.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "displayapp/screens/Screen.h"
+#include "displayapp/widgets/Counter.h"
+#include "components/datetime/DateTimeController.h"
+#include <lvgl/lvgl.h>
+
+namespace Pinetime::Applications::Screens {
+  class Dice : public Screen {
+    public:
+      Dice(DisplayApp* app, Controllers::DateTime& dateTime);
+      ~Dice() override;
+      void Roll();
+      void NextColor();
```

Comment:
the `NextColor()` function is a private helper. I don't think it is needed in the public interface of the App. please move into the private namespace

Reply:
addressed in b1e522fc42e8819e7d8ca90e7a8e2b877f6fd303